bin/runtime/src/lib.rs

```diff
@@ -260,7 +260,7 @@ impl_opaque_keys! {
 }
 
 parameter_types! {
-    pub const Period: u32 = 50;
+    pub const Period: u32 = 5;
```

Comment:
Do we want period in final version to be so small?

Reply:
Honestly, I don't think it matters -- this is typically run only for testing purposes, where you want to have short sessions, to catch bugs with session change. For actual testnet, this is changed anyway, and we will soon have this field as part of a config file.